systest/Makefile

```
@@ -41,22 +42,26 @@ docker:
 push:
 	docker push $(image_name)
 
+.PHONY: template
+template:
+	@echo smesher=`cat $(smesher_config)` >> $(tmpfile)
+	@echo poet=`cat $(poet_config)` >> $(tmpfile)
```

There is a risk of getting an invalid file if it just appends, when the file already exists. Consider writing on the first `echo`, or removing the file:
```suggestion
.PHONY: template
template:
	@echo smesher=`cat $(smesher_config)` > $(tmpfile)
	@echo poet=`cat $(poet_config)` >> $(tmpfile)
```

---

i am removing a file after reading it in kubectl, see config subcommand
this is the best i could come up with. do you know how it can be done better?

---

actually didnt notice, changing >> to > is neat. will use